packages/viewer/src/components.d.ts

```diff
@@ -1071,6 +1088,10 @@ export namespace Components {
      * This property will automatically be set when a child of a `<vertex-viewer-markup>` element.
      */
     disabled: boolean;
+    /**
+     * The line end style of the ending anchor. This default to 'arrow-triangle.'
+     */
+    endLineEndStyle: LineEndStyle;
```

Comment:
nit: are there any other naming conventions we could explore besides `EndStyle`? I don't love the word `end` used twice here.

A few ideas:
- `endLineStyle`
- `endLineTipStyle`
- `endLineAnchorType`